backend/logic/courses/courses.js

```diff
@@ -0,0 +1,9 @@
+const mysql = require('../../sql/connection');
+
+exports.queryCoursesByTag = (tag) => {
+    return new Promise(function(resolve,reject){
+        let promise = new Promise(function(resolve,reject){
```

Comment:
I like that you are using promises, thats good! I think its important to know why you used promises here. Like for instance, you have promises with the resolve and rejection, so feel free to use them to indicate how your code will proceed after the completion of an asynchronous operation

Reply:
Oh my bad, this is incomplete code that is completed in my other PR

Reply:
Thanks for catching this, it will however be resolved after the other one is merged